src/lib/instanceManager/createInstanceManager.js

```diff
@@ -22,6 +22,33 @@ module.exports = ({
   const { instances: instancesSettings } = turbine.getExtensionSettings();
   const instanceByName = {};
 
+  const calledMonitors = {};
```

Comment:
This object represents a memory leak. I wasn't worried before when we were talking about just instanceCreated, and instanceConfigured, but now with more methods that will get called as the page progresses, this could represent a memory leak especially if someone called evaluateRulesets on a regular cadence. Maybe after 10 seconds we should remove the monitor that just records to calledMonitors. I was looking for an event in Tags that would be called when everything is initialized, but I couldn't find one.